lib/telephony/test/voice_sender.rb

```diff
@@ -1,15 +1,30 @@
+# frozen_string_literal: true
+
 module Telephony
   module Test
     class VoiceSender
+      ORIGINATION_PHONE_NUMBER = '+1888LOGINGOV'
```

Comment:
It is not a valid US number (11 digits instead of 10), but we shouldn't be trying to parse this as one, so I figure it's OK as-is? open to suggestions